src/components/sidebar/sidebar.stories.tsx

```diff
@@ -315,3 +316,33 @@ export const WithHeaderAndFooterPadding: ComponentStory<
     </>
   );
 };
+
+export const TopModalOverride = () => {
```

Comment:
**suggestion**: this story *really* needs a button to kick everything off, like the ones you've added for Dialog/DialogFullScreen do. Otherwise we're going to really annoy anyone who goes to our docs for Sidebar, who likely don't care at that time about `topModalOverride`. To add insult to injury, if they actually do, by the time they've scrolled down to it and read it, there's no way to retrigger the example short of refreshing the page!